cumulusci/tasks/bulkdata/load.py

```diff
@@ -110,7 +110,7 @@ def _run_task(self):
     def _load_mapping(self, mapping):
         """Load data for a single step."""
 
-        if "RecordTypeId" in mapping["fields"]:
+        if mapping.get("fields", {}).get("RecordTypeId"):
```

Comment:
Yes, but so far I have left the code so that it is compatible with raw dictionaries and also with the new DictModels. Then I can test "expected" behaviour against "new" behaviour by changing a single line of code.

The long-term plan should be to move away form dict syntax altogether IMO. And yes, lots of defaults.

Reply:
Okay, sounds like a reasonable approach.